tests/pageinfo-records.test.js

```diff
@@ -24,6 +24,11 @@ test("run warc and ensure pageinfo records contain the correct resources", async
   let foundInvalid = false;
 
   for await (const record of parser) {
+    if (record.warcType === "response") {
+      expect(record.warcHeaders.headers.get("WARC-Protocol")).toBe("h2, tls/1.3");
```

Comment:
Seeing a test failure here. Getter might not be working as expected due to repeated headers now?